HTML/mockup-mogo/js/task6.js

```diff
@@ -0,0 +1,27 @@
+import {loadedElements} from './elements.js';
+
+export function displayAccordion() {
+  loadedElements.then((elements) => {
+    elements.optionsPart.addEventListener('click', (e) => {
+      let targetClosestOption = e.target.closest('.option-item__top');
+
+      if (!targetClosestOption) {
+        return false;
+      }
+
+      let targetIndex = targetClosestOption.dataset.index;
+
+      elements.optionTop.forEach((el, index) => {
+        if (index == targetIndex) {
+          elements.paragraph[index].classList.add('u-show');
+          elements.arrow[index].src = 'img/arrow-up@3x.png';
```

Comment:
it is better to toggle class that change src

Reply:
or have both images on the page and trigger `display: none` by class

Reply:
hm. not sure. because there will be latency right after - browser doesn't load images that are hidden by `display: none`

Reply:
ah, no. Both images will be already downloaded because it's displayed since start